docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -21,64 +21,56 @@ tags:
 
 :::tip Support, stability, and dependency info
 
-- In [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
-
-:::
+- API Keys used for Cloud Operations Tools (tcld, API, Terraform) are in [Public Preview](/evaluate/development-production-features/release-stages#public-preview)
```

Comment:
This is the overall structure of your document.

```
## Overview
## Get started {#get-started}
### Prerequisites
## API Key Management and Governance {#manage-api-keys}
### Global Administrator API Key Management {#ga-manage-api-keys}
## User API Keys {#user-api-keys}
### Generate an API Key
#### Using the Cloud UI
#### Using tcld
### Delete an API Key
#### Using the Cloud UI
#### Using tcld
### Enable/Disable an API Key
#### Using the Cloud UI
#### Using tcld
## Managing API Keys for Service Accounts {#serviceaccount-api-keys}
### Create an API Key for a Service Account
#### Using the Cloud UI
#### Using tcld
### Enable/Disable an API Key
#### Using the Cloud UI
#### Using tcld
### Delete an API Key for a Service Account
#### Using the Cloud UI
#### Using tcld
## API Keys for Namespace Authentication {#namespace-authentication}
## Using API Keys {#using-apikeys}
### Temporal CLI
### SDK
#### Go SDK (pre v 1.26.0)
#### Java SDK
#### Python SDK (v 1.6.0+)
#### Python SDK (pre v 1.6.0)
#### .NET SDK (v 1.1.0+)
### tcld
### The Cloud Ops API
### The Terraform Provider
## Best practices {#best-practices}
## Troubleshooting {#troubleshooting}
## Frequently asked questions {#faqs}
```

From the style guide:

> Use H3 headers sparingly, and avoid H4 headers. If you need to use subheaders, make sure there are two or more headers of that level within that section of the tutorial. Alternatively, consider making multiple steps.

There are several places that do not follow the balance rule for "at least two" H3 or lower headings in this doc.